features/org.wso2.micro.integrator.server.feature/pom.xml

```diff
@@ -272,6 +272,13 @@
                                 <bundleDef>org.atomikos.wso2:atomikos:${atomikos.version}</bundleDef>
                                 <bundleDef>org.eclipse.platform:org.eclipse.equinox.cm:${version.eclipse.equinox.cm}</bundleDef>
                                 <bundleDef>commons-configuration:commons-configuration:${commons.configuration.version}</bundleDef>
+                                <!-- bundles needed to swagger for DSS feature -->
+                                <bundleDef>io.swagger:swagger-models:${io.swagger.models.version}</bundleDef>
+                                <bundleDef>io.swagger:swagger-core:${io.swagger.core.version}</bundleDef>
+                                <bundleDef>io.swagger:swagger-annotations:${io.swagger.annotation.version}</bundleDef>
+                                <bundleDef>com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${jackson.dataformat.yaml.version}</bundleDef>
+                                <bundleDef>javax.validation:validation-api:${javax.validation.version}</bundleDef>
+                                <bundleDef>org.yaml:snakeyaml:${snakeyaml.version}</bundleDef>
```

Comment:
Shouldn't we bundle these inside the respective feature itself? @Nirothipan

Reply:
+1 , It seems this is used in API and in DSS  hence it has been bundled here. However  we should be bundling in both places if required.

Reply:
We can add these bundles in the DSS feature or MI feature. I chose MI feature since its where the request processors reside and its where these dependency jars are used.